cmd/libs/go2idl/types/types.go

```diff
@@ -275,6 +275,18 @@ var (
 		Name: Name{Name: "uint"},
 		Kind: Builtin,
 	}
+	Float64 = &Type{
```

Comment:
My thought exactly :)

On Nov 2, 2015, at 4:05 PM, Daniel Smith <notifications@github.com> wrote:

In cmd/libs/go2idl/types/types.go
<https://github.com/kubernetes/kubernetes/pull/16649#discussion_r43681965>:

> @@ -275,6 +275,18 @@ var (
>  		Name: Name{Name: "uint"},
>  		Kind: Builtin,
>  	}
> +	Float64 = &Type{

Wow, I totally forgot about floats! We can keep pretending complex types
don't exist ;)

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/16649/files#r43681965>.